engine/env/spark/src/main/java/org/datacleaner/spark/utils/HadoopUtils.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.spark.utils;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.metamodel.MetaModelException;
+import org.apache.metamodel.util.FileHelper;
+
+import com.google.common.base.Strings;
+
+public class HadoopUtils {
```

Comment:
Maybe we should call this something like HadoopClientUtils because it is really only valid for cases where there is a namenode (e.g. not MapR) or so I think ... At least my experiments so far failed to show that a client on MapR could create a configuration like this. But this stuff works with "vanilla" Hadoop and also Hortonworks - maybe also Cloudera.